src/js/common/schemaform/SaveInProgressErrorPage.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import { LOAD_STATUSES, fetchInProgressForm, setFetchFormStatus } from './save-load-actions';
+import SignInLink from '../components/SignInLink';
+import ProgressButton from '../components/form-elements/ProgressButton';
+
+import { updateLogInUrl } from '../../login/actions';
+
+
+// For now, this only handles loading errors, but it could feasibly be reworked
+//  to handle save errors as well if we need it to.
+class SaveInProgressErrorPage extends React.Component {
+  getBackButton = (primary = false) => {
+    const buttonClass = primary ? 'usa-button-primary' : 'usa-button-outline';
+    const goBack = () => {
+      this.props.setFetchFormStatus(LOAD_STATUSES.notAttempted);
+      this.props.router.goBack();
+    };
+    return (
+      <ProgressButton
+          onButtonClick={goBack}
+          buttonClass={buttonClass}
+          buttonText="Back"
+          beforeText="«"/>
+    );
+  }
+
+  // Reload the form and try again.
+  reloadForm = () => {
+    // formConfig is put in this.props.routes[length - 1]
+    const formConfig = this.props.routes[this.props.routes.length - 1].formConfig;
+    this.props.fetchInProgressForm(formConfig.formId, formConfig.migrations);
+  }
+
+  render() {
+    const { loadedStatus } = this.props;
+    const { noAuth, notFound } = this.props.errorMessages || {};
```

Comment:
It looks like these aren't coming through correctly, I see a truncated sign in message when running this.